STYLE-GUIDE.md

```diff
@@ -19,10 +19,28 @@ Words are important.  Pulumi strives to use language that is clear, harmonious,
 * Ensure that readers are able to scan the headings of a page and get an effective overview of the page's content.
 * Every page should have exactly one `h1`.
 * Headings levels should only increment one level at a time.  E.g., if your previous heading level was an `h2`, the next heading must be an `h2` or an `h3`, but not, e.g., an `h4` or `h5`.
+*  Include keywords relating to the content immediately following the heading in `h2` and deeper.
+*  Try to front-load your keywords into the sentence to more immediately indicate the subject matter of the following content.
+*  - Alternate your topical keywords, visit the [online Thesarus](https://www.thesaurus.com/) if you are stuck.
+*  Try not to use repetitive headings (e.g., starting with the same word, using the same keyword)
+
+## Images
+
+* Include a minimum of one image that supports your topic beneath the first `h2`. 
```

Comment:
are you suggesting that every marketing page, docs page, and registry page should have an image in this location? or is this only for the blogs? if only for the blogs id encourage moving this information to a blogs section, or to the blog markdown file.

Reply:
Every page should have one topically relevant image, in the registry and docs, it's ok to use the logo of the product in question.

Reply:
I'd characterize this as a more of a recommendation than a requirement -- and assuming it's SEO-relavant, explain how. Practically speaking, it's just not the kind of thing we're going to be able to do in all cases, and even if we could, I'm not sure it'd always make sense, content-wise.

Reply:
@Brant-Soler Could we maybe put SEO considerations in a separate section along with a list of content types they apply to (blog posts, etc.)? I figure that these recommendations are not intended to apply to e.g. docs for individual resources/functions in the Registry. If the intent is to make significant changes to our detailed technical docs for SEO, then that seems like a larger change that warrants a wider discussion.

Reply:
Agreed on both counts, there are certainly more SEO-related recommendations. Let me take a pass at the phrasing and get your opinion on striking the section and moving it to a new location or leaving it be. Another consideration with the images that should be built into every page is having an image to represent the link on social media when shared. Would you recommend that go here or to another area?